internal/configs/module_call_test.go

```diff
@@ -50,17 +50,17 @@ func TestLoadModuleCall(t *testing.T) {
 			SourceAddr: addrs.ModuleSourceRegistry{
 				Package: addrs.ModuleRegistryPackage{
 					Host:         addrs.DefaultModuleRegistryHost,
-					Namespace:    "hashicorp",
+					Namespace:    addrs.DefaultProviderNamespace,
```

Comment:
DefaultModuleNamespace?

Reply:
This shouldn't actually change because modules shouldn't change?